libraries/launcher/org/prismlauncher/utils/logging/LogPrintStream.java

```diff
@@ -33,48 +33,67 @@
  *  along with this program.  If not, see <https://www.gnu.org/licenses/>.
  */
 
-package org.prismlauncher;
+package org.prismlauncher.utils.logging;
 
-import org.prismlauncher.impl.OneSixLauncher;
-import org.prismlauncher.utils.Parameters;
+import java.io.OutputStream;
+import java.io.PrintStream;
 
-import java.util.HashMap;
-import java.util.Map;
+/**
+ * Used to create a print stream that redirects to Log.
+ */
+final class LogPrintStream extends PrintStream {
```

Comment:
I don't like this. Don't we have another way of doing this?

Reply:
There are alternatives, but they would likely miss some of the nuances of the stock implementation, which is fairly complex.

Reply:
From what I can tell this is only used for stacktraces. I think it would make everyone's life easier if stacktraces don't have any prefix anyway.

Reply:
The prefixes are used to signal the text colour to the log viewer.
The log viewer *does* automatically convert stacktraces to red though.
Just means the fatal stacktraces won't get the black background.

Reply:
IMO it's fine. A fatal stacktrace generally means that the game/launcher will quit immediately. So it should be fine